projects/demo/src/modules/components/input-date-range/input-date-range.template.html

```diff
@@ -19,6 +19,28 @@
         >
             <tui-input-date-range-example-2></tui-input-date-range-example-2>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="date-localization"
+            heading="Date localization"
+            [content]="example3"
+        >
+            <h3 i18n>DI-tokens for date format localization:</h3>
```

Comment:
nit: shouldn't we move the customization description from the concrete example to general description? Just like in dialog, input-number, swipe etc

Reply:
done